internal/pkg/consensus/testing.go

```diff
@@ -196,62 +197,57 @@ func NFakeSectorInfos(numSectors uint64) ffi.SortedPublicSectorInfo {
 	return ffi.NewSortedPublicSectorInfo(infos...)
 }
 
-// SeedFirstWinnerInNRounds returns a ticket that when mined upon for N rounds
+// SeedFirstWinnerInNRounds returns seeded fake chain randomness that when mined upon for N rounds
 // by a miner that has `minerPower` out of a system-wide `totalPower` and keyinfo
 // `ki` will produce a ticket that gives a winning election proof in exactly `n`
 // rounds.  No wins before n rounds are up.
 //
 // Note that this is a deterministic function of the inputs as we return the
-// first ticket that seeds a winner in `n` rounds given the inputs starting from
-// MakeFakeTicketForTest().
+// first such seed.
 //
 // Note that there are no guarantees that this function will terminate on new
 // inputs as miner power might be so low that winning a ticket is very
 // unlikely.  However runtime is deterministic so if it runs fast once on
 // given inputs is safe to use in tests.
-func SeedFirstWinnerInNRounds(t *testing.T, n int, ki *crypto.KeyInfo, networkPower, numSectors, sectorSize uint64) block.Ticket {
-	tm := TicketMachine{}
+func SeedFirstWinnerInNRounds(t *testing.T, n int, miner address.Address, ki *crypto.KeyInfo, networkPower, numSectors, sectorSize uint64) *FakeChainRandomness {
```

Comment:
This stuff was brain-wracking to adapt to the new world. The tests are skipped so I'm not 100% sure this works.

Reply:
Yeah looks like it.  Thanks for heads up.